### ISSUE TITLE
Development protocol

### ISSUE THREAD

**Issue by pakozm** [april-org/april-ann#39]
1. Every feature has to be developed in a new branch. Contributors can have its own fork and work in new branches for new features.
2. Once a feature is finished, a pull requests into `devel` branch is mandatory, unless commits with bug or memory leak problems solving which can be done directly into `devel` without a pull request. Please, avoid to do merge of feature branches into `devel`, a pull request allow other developers and contributors to talk about the feature before it is merged into `deve`.
3. After the pull request, the branch must to be deleted.
4. Every friday, the stable version in `devel` branch will be merged with `master` branch.
5. Every friday, after previous step, your stable changes could be committed to the branch `devel`.
6. During the week, `devel` branch must be checked and mark as stable.

Pushes to devel are throwing a travis process to ensure the good compilation and testing the tool. The same is configured to pushes and pull requests to master branch.
